.travis.yml

```diff
@@ -11,15 +9,23 @@ python:
   # - "3.6-dev"
   # - "3.7-dev"
 
-services:
-  - postgresql
+sudo: required
 
-before_script:
-  - psql -c 'create database travis_ci_test;' -U postgres
+dist: xenial        
+        
+services:
+    - docker
 
 install:
-        #- curl -sL git.io/install-candis | sudo python
-  - sudo python get-candis
+    - docker-compose build
+    - docker-compose up -d
+    - docker-compose exec app sh -c "apt-get update && apt-get install -y netcat"
+
 script:
-  - redis-cli ping
-  - make test
+    - docker-compose exec app sh -c "nc -vz cache 6379"
+    - docker-compose exec app sh -c "nc -vz db 5432"
+    - docker-compose exec app sh -c "nc -vz proxy 80"
```

Comment:
Very good. Can you create an API route that does a PING?

Something like.

```
@app.route('/ping')
def ping():
     return "{ 'data': 'pong' }"
```

Then maybe, test this route.

`80` checks the proxy. You might also need to check the internal app working.

And then something like

```
$ docker-compose exec proxy sh -c "curl https://localhost/api/ping"
```

Reply:
Yeah right. @achillesrasquinha so should I write the ping function in a new file in `/candis/api/server/api/ping.py` or do it in `/candis/api/server/api/user.py` .

Reply:
@achillesrasquinha  Done.